### DISCUSSION TITLE
Manually Install this plugin for Qidi Slicer?

### DISCUSSION THREAD

**Discussion by supared25** [5axes/Calibration-Shapes#137] [Q&A]
Qidi Slicer is a version of Cura, except there is no Marketplace. Any tips on how I can install this plugin manually on Qidi. I imagine if there is a way to do it manually with Cura, then it can be one with Qidi the same way.

**Comment by 5axes** [comment]
Not sure of that, but Qidi has also rename all the UM reference in their code, so the source code is not compatible with this slicer.

**Reply by 5axes** [reply]
Ok I confirm, every UM or ultimaker reference have been change to QD or qidi. So even if Qidi printer is based on the Cura source code the macro or plugin developped for cura cannot be used in this slicer...

**Reply by supared25** [reply]
Thank you for checking!
